doc/src/install.rst

```diff
@@ -8,26 +8,115 @@ Installation
 Psycopg is a PostgreSQL_ adapter for the Python_ programming language. It is a
 wrapper for the libpq_, the official PostgreSQL client library.
 
-The `psycopg2` package is the current mature implementation of the adapter: it
```

Comment:
For someone new to the project/python, this section might be a confusing way to start. I moved it down to the _Non-standard Builds_ section